File: alienvolwels.c
#include <stdio.h>
#include <string.h>

int main(){

    char vogais[100];
    char entrada[100];
    int quantidadeVogais=0;

    scanf ("%s", vogais);
    scanf (" %[^\n]", entrada);

    int tam=strlen(entrada);

    for (int i=0;i<tam; i++){

        char caractereAtual= entrada[i];

        for (int j=0;j<strlen(vogais);j++){

            if (caractereAtual== vogais[j]){

                quantidadeVogais++;
                break;
            }
        }
    }

    printf ("%d", quantidadeVogais);
    return 0;
}

File: alienvolwels.java
import java.util.Scanner;

public class alienvolwels{


    public static void main (String[] args){

        Scanner scanner= new Scanner(System.in);

        String vogais= scanner.nextLine();

        char[] vetorDeCaracteres = vogais.toCharArray();

        String frase= scanner.nextLine();

        int quantidadevogais=0;
        for (int i=0;i<frase.length();i++){

            char caractereAtual= frase.charAt(i);
            for (int j=0;j<vetorDeCaracteres.length;j++){

                if (caractereAtual==vetorDeCaracteres[j]){

                    quantidadevogais++;
                }
            }
        }

        System.out.println(quantidadevogais);
       
    }
}